doc/rtmpwebhookauth.md

```diff
@@ -1,24 +1,28 @@
 # RTMP Webhook Authentication
 
-Incoming RTMP streams can be authenticated using Webhooks. To use it you need to implement own web server which will be called by Livepeer node when new RTMP stream received to check if that stream is allowed.
+Incoming RTMP streams can be authenticated using Webhooks. To provide these webhooks, node operators must implement their own web service / endpoint to be accessed only by the Livepeer node. As new RTMP streams appear, the Livepeer node will hit this endpoint to determine whether the given stream is allowed.
 
-To enable Webhook authentication functionality Livepeer node should be started with flag `-authWebhookUrl`, like this:
+To enable webhook authentication functionality, the Livepeer node should be started with the flag, `-authWebhookUrl` along with the webhook endpoint URL.
 
-`livepeer -authWebhookUrl http://ownserver/auth`
+For example:
 
-For every incoming RTMP stream Livepeer node will make `POST` request to `http://ownserver/auth` endpoint passing url to which RTMP request was made as JSON object.
+```console
+livepeer -authWebhookUrl http://ownserver/auth
+```
+
+For each incoming RTMP stream, the Livepeer node will make a `POST` request to the `http://ownserver/auth` endpoint, passing the URL of the RTMP request as JSON object.
 
-For example, if incoming RTMP request was made to `rtmp://livepeer.node:1935/something?manifestID=manifest`, Liverpeer node will pass this object in request to webhook:
+For example, if the incoming RTMP request was made to `rtmp://livepeer.node:1935/something?manifestID=manifest`, the Liverpeer node will provide the following object as a request to the webhook endpoint:
```

Comment:
The RTMP URL can be changed to `rtmp://livepeer.node:1935/manifest` to match the json body. Also, the `:1935` can probably be omitted since it's implied as the default RTMP port (json body example should be updated as well).

Reply:
done in f69f993fffbd8bc2d3c6eb5e0a600e64ad420e5e